lib/__tests__/breadcrumbs.test.ts

```diff
@@ -27,31 +30,37 @@ describe("Test breadcrumb composition and rendering functionality", () => {
       })
     );
 
-    const labItemData = {
+    const labItemData: DatabaseObject = {
+      "@context": "/terms/",
       name: "j-michael-cherry",
       "@id": "/labs/j-michael-cherry/",
       "@type": ["Lab", "Item"],
       title: "J. Michael Cherry, Stanford",
+      status: "current",
+      uuid: "d91b048e-2d8a-4562-893d-93f0e68397c0",
     };
 
-    let breadcrumbs = await buildBreadcrumbs(labItemData, labItemData.title);
```

Comment:
Why `labItemData.title` is not working anymore in typescript?

Reply:
Yeah, I didn’t use the right solution there. I should have typecast the argument, which I now do. Really weird that Typescript can’t see that the `title` property is a string though. I assign a string to `title`, and Typescript thinks that’s a mystery?

Reply:
I guess Typescript is not that clever or just very strict...